scalding-beam/src/test/scala/com/twitter/scalding/beam_backend/BeamBackendTests.scala

```diff
@@ -0,0 +1,60 @@
+package com.twitter.scalding.beam_backend
+
+import com.twitter.scalding.{Config, Execution, TextLine, TypedPipe}
+import java.io.File
+import java.nio.file.Paths
+import org.apache.beam.sdk.options.{PipelineOptions, PipelineOptionsFactory}
+import org.scalatest.{BeforeAndAfter, FunSuite}
+
+class BeamBackendTests extends FunSuite with BeforeAndAfter {
+
+  private var pipelineOptions: PipelineOptions = _
+
+  def tmpPath(suffix: String): String = {
+    Paths.get(
+      System.getProperty("java.io.tmpdir"),
+      "scalding",
+      "beam_backend",
+      suffix).toString
+  }
+
+  before {
+    pipelineOptions = PipelineOptionsFactory.create()
+  }
+
+  test("test") {
+    val outRoute1 = tmpPath("out1")
+    val outRoute2 = tmpPath("out2")
+
+    val pipe1 = TypedPipe
+      .from(0 to 100)
+      .map(x => (x, x))
+      .mapValues(x => x * 3)
+      .filterKeys(x => x % 2 == 0)
+      .map(_.toString())
+    val ex1 = pipe1.writeExecution(TextLine(outRoute1))
+
+    val pipe2 = TypedPipe
+      .from(1 to 20)
+      .map(x => (x, x))
+      .flatMapValues(x => 1 to x)
+      .map(_.toString())
+    val ex2 = pipe2.writeExecution(TextLine(outRoute2))
+
+    val bmode = BeamMode.default(pipelineOptions)
+    val ex3 = Execution.zip(ex1, ex2)
+    ex3.waitFor(Config.empty, bmode).get
+
```

Comment:
Can we open the files using normal Java IO and make some assertion about them?

Like read all the numbers and put them in a set and assert it matches the expected Set?

I'd like a bit more assurance than it just didn't crash.